torch/distributed/distributed_c10d.py

```diff
@@ -172,6 +173,22 @@ class GroupMember(object):
 # Process group count for default naming
 _group_count = 0
 
+STORE_BASED_BARRIER_PREFIX = "store_based_barrier_key"
+
+def _store_based_barrier(rank, store, timeout):
+    global _group_count
```

Comment:
Looks like we are only reading from `_group_count`. If that's the case, shall we drop this line?